scripts/update-app-download-links.js

```diff
@@ -26,6 +28,6 @@ module.exports = function updateAppDownloadLinks () {
   }
 
   if (links.length > 1) {
-    document.querySelector('#view-all-downloads').style.display = 'inline-block'
+    document.querySelector('#view-all-downloads').style.display = 'block'
```

Comment:
`Download` section would be empty if there's only 1 available distribution that is not matching current OS.

Reply:
The download section is not displayed unless there are downloads, because there's a conditional in the handlebars template: https://github.com/electron/electronjs.org/blob/cff9f468999d018d0505ee1dbf14b6155df17635/views/apps/show.html#L92-L93

Reply:
That's what I expected to happen, but I don't think this is the case? (see https://electronjs.org/apps/protegopdf for an example)

Reply:
Doh! Opened https://github.com/electron/electronjs.org/issues/1221